python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -1366,6 +1366,72 @@ def testPureOSLShaders( self ) :
 			self.assertEqual( arnold.AiNodeEntryGetName( arnold.AiNodeGetNodeEntry( noise ) ), "osl" )
 			self.assertEqual( arnold.AiNodeGetStr( noise, "shadername" ), "Pattern/Noise" )
 
+	def testOSLMultipleOutputs( self ) :
+
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"Arnold",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.SceneDescription,
+			self.temporaryDirectory() + "/test.ass"
+		)
+
+		network = IECore.ObjectVector( [
+			IECoreScene.Shader(
+				"Utility/SplitColor",
+				"osl:shader",
+				{
+					"c" : imath.Color3f( 0.1, 0.2, 0.3 ),
+					"__handle" : "splitHandle"
+				}
+			),
+			IECoreScene.Shader(
+				"Utility/BuildColor",
+				"osl:shader",
+				{
+					"r" : "link:splitHandle.r",
+					"g" : "link:splitHandle.g",
+					"b" : "link:splitHandle.b",
+				}
+			)
+		] )
+
+		o = r.object(
+			"testPlane",
+			IECoreScene.MeshPrimitive.createPlane( imath.Box2f( imath.V2f( -1 ), imath.V2f( 1 ) ) ),
+			r.attributes( IECore.CompoundObject( { "ai:surface" : network } ) )
+		)
+		del o
```

Comment:
If we just don't assign to `o` in the first place, then we wouldn't have to `del o` here, since we don't use `o` for anything.